cache-config/testing/ort-tests/t3c-reload_test.go

```diff
@@ -26,226 +26,163 @@ import (
+		t.Fatalf("failed to remove file '%s': %v", fileNameToRemove, err)
 	}
 
-	t.Logf("DEBUG doTestT3cReloadHeaderRewrite setting update flag")
 	// set the update flag, so syncds will run
-	if err := ExecTOUpdater("atlanta-edge-03", false, true); err != nil {
-		t.Fatalf("t3c-update failed: %v\n", err)
+	if err := ExecTOUpdater(cacheHostName, false, true); err != nil {
+		t.Fatalf("t3c-update failed: %v", err)
 	}
 
-	t.Logf("DEBUG doTestT3cReloadHeaderRewrite calling syncds")
 	stdOut, _ := t3cUpdateReload(cacheHostName, "syncds")
 	// Ignore the exit code error for now, because the ORT Integration Test Framework doesn't currently start ATS.
 	// TODO check err, after running ATS is added to the tests.
 	// if err != nil {
-	// 	t.Fatalf("t3c syncds failed: %v\n", err)
+	// 	t.Fatalf("t3c syncds failed: %v", err)
 	// }
 
-	t.Logf("DEBUG TestT3cReload looking for reload string")
 	if !strings.Contains(stdOut, `Running 'traffic_ctl config reload' now`) {
-		t.Errorf("expected t3c to do a reload after adding a etc/trafficserver file, actual: '''%v'''\n", stdOut)
+		t.Errorf("expected t3c to do a reload after adding a etc/trafficserver file, actual: %s", stdOut)
 	}
-
-	t.Logf("------------- End TestT3cReload doTestT3cReloadAnythingInTrafficserverDir ---------------")
 }
 
 func doTestT3cReloadNoChange(t *testing.T) {
-	t.Logf("------------- Start doTestT3cReloadNoChange ---------------")
-
-	cacheHostName := "atlanta-edge-03"
-
-	t.Logf("DEBUG doTestT3cReloadHeaderRewrite calling badass")
 	if stdOut, exitCode := t3cUpdateReload(cacheHostName, "badass"); exitCode != 0 {
-		t.Fatalf("ERROR: t3c badass failed: code '%v' output '%v'\n", exitCode, stdOut)
+		t.Fatalf("t3c badass failed with exit code %d, output: %s", exitCode, stdOut)
 	}
 
-	t.Logf("DEBUG doTestT3cReloadHeaderRewrite deleting file")
-
 	// no change, should not trigger a reload
 
-	t.Logf("DEBUG doTestT3cReloadHeaderRewrite setting update flag")
 	// set the update flag, so syncds will run
-	if err := ExecTOUpdater("atlanta-edge-03", false, true); err != nil {
-		t.Fatalf("t3c-update failed: %v\n", err)
+	if err := ExecTOUpdater(cacheHostName, false, true); err != nil {
+		t.Fatalf("t3c-update failed: %v", err)
 	}
 
-	t.Logf("DEBUG doTestT3cReloadHeaderRewrite calling syncds")
 	stdOut, _ := t3cUpdateReload(cacheHostName, "syncds")
 	// Ignore the exit code error for now, because the ORT Integration Test Framework doesn't currently start ATS.
 	// TODO check err, after running ATS is added to the tests.
 	// if err != nil {
-	// 	t.Fatalf("t3c syncds failed: %v\n", err)
+	// 	t.Fatalf("t3c syncds failed: %v", err)
 	// }
 
-	t.Logf("DEBUG TestT3cReload looking for reload string")
 	if strings.Contains(stdOut, `Running 'traffic_ctl config reload' now`) {
-		t.Errorf("expected t3c to not reload after no change, actual: '''%v'''\n", stdOut)
+		t.Errorf("expected t3c to not reload after no change, actual: %s", stdOut)
```

Comment:
The t3c output here is very large, and contains many many newlines. I think the triple-quotes add a lot of value, making it easy to see where this particular output begins and ends, versus any other errors.

Reply:
> It also also also re-structures some log messages so that triple-quoting
> possibly multi-line strings is unnecessary - because now they're at the
> end of the log line, so they won't be confused with the rest of it.

But there might be multiple errors, the quotes help distinguish between other errors, with this and other tests. We're talking over a hundred lines here, it's hard to even find where the next test begins. (But it's necessary, without the output it maybe impossible to figure out why the failure occurred.)

Reply:
I introduced a few test failures to check out the output in the event of a failure. Specifically, I applied this diff:
```diff
diff --git a/cache-config/testing/ort-tests/t3c-reload_test.go b/cache-config/testing/ort-tests/t3c-reload_test.go
index d04107299..bb79be831 100644
--- a/cache-config/testing/ort-tests/t3c-reload_test.go
+++ b/cache-config/testing/ort-tests/t3c-reload_test.go
@@ -92,7 +92,7 @@ func doTestT3cReloadAnythingInTrafficserverDir(t *testing.T) {
        //      t.Fatalf("t3c syncds failed: %v", err)
        // }
 
-       if !strings.Contains(stdOut, `Running 'traffic_ctl config reload' now`) {
+       if !strings.Contains(stdOut, `Running 'traffic_ctl config reload' now4614155724`) {
                t.Errorf("expected t3c to do a reload after adding a etc/trafficserver file, actual: %s", stdOut)
        }
 }
@@ -116,12 +116,14 @@ func doTestT3cReloadNoChange(t *testing.T) {
        //      t.Fatalf("t3c syncds failed: %v", err)
        // }
 
-       if strings.Contains(stdOut, `Running 'traffic_ctl config reload' now`) {
+       t.Error("not a real error")
+       if strings.Contains(stdOut, `Running 'traffic_ctl config reload' now1114515`) {
                t.Errorf("expected t3c to not reload after no change, actual: %s", stdOut)
        }
 }
 
 func doTestT3cRevalCallsReload(t *testing.T) {
+       t.Error("not a real error")
        if stdOut, exitCode := t3cUpdateReload(DefaultCacheHostName, "badass"); exitCode != 0 {
                t.Fatalf("t3c badass failed with exit code %d, output: %s", exitCode, stdOut)
        }
```
That way there's a failure in a sub-test immediately preceding and following this sub-test, and a failure within the same sub-test immediately preceding this test failure. The logs can be found below

<details><summary>Test Logs</summary>

```
Config: &{{https://to_server:443 twelve {disallowed readOnly operations admin portal federation extension} false} {traffic_ops db traffic_ops twelve 5432 Pg false Test database to_test} {{60} {stdout stdout stdout stdout stdout} false} true}
INFO: traffic_ops_ort_test.go:88: 2021-11-23T16:26:06.985676192Z: Using Config values:
	TO Config File:              conf/docker-edge-cache.conf
	TO Fixtures:                 tc-fixtures.json
	TO URL:                      https://to_server:443
	TO Session Timeout In Secs:  60
	DB Server:                   db
	DB User:                     traffic_ops
	DB Name:                     traffic_ops
	DB Ssl:                      false
	UseIMS:                      true
--- FAIL: TestT3cReload (18.37s)
    cdns.go:26: Response:  {[{cdn was created. success}]}
    cdns.go:26: Response:  {[{cdn was created. success}]}
    cdns.go:26: Response:  {[{cdn was created. success}]}
    cdns.go:26: Response:  {[{cdn was created. success}]}
    cdns.go:26: Response:  {[{cdn was created. success}]}
    types.go:24: ---- CreateTestTypes ----
    profile_parameters.go:50: Response:  {[{profileParameter was created. success}]}
    divisions.go:25: Response:  {[{division was created. success}]}
    divisions.go:25: Response:  {[{division was created. success}]}
    regions.go:27: Response:  {[{region was created. success}]}
    regions.go:27: Response:  {[{region was created. success}]}
    phys_locations.go:25: Response:  {[{physLocation was created. success}]}
    phys_locations.go:25: Response:  {[{physLocation was created. success}]}
    phys_locations.go:25: Response:  {[{physLocation was created. success}]}
    topologies.go:48: Response: {Response:{Description:a multi-site origin topology Name:mso-topology Nodes:[{Id:0 Cachegroup:multiOriginCachegroup Parents:[] LastUpdated:<nil>} {Id:0 Cachegroup:parentCachegroup Parents:[0] LastUpdated:<nil>} {Id:0 Cachegroup:cachegroup2 Parents:[1] LastUpdated:<nil>}] LastUpdated:<nil>} Alerts:{Alerts:[{Text:topology was created. Level:success}]}}
    topologies.go:48: Response: {Response:{Description:another topology Name:another-topology Nodes:[{Id:0 Cachegroup:parentCachegroup Parents:[] LastUpdated:<nil>} {Id:0 Cachegroup:cachegroup1 Parents:[0] LastUpdated:<nil>} {Id:0 Cachegroup:secondaryCachegroup Parents:[] LastUpdated:<nil>} {Id:0 Cachegroup:cachegroup2 Parents:[2] LastUpdated:<nil>}] LastUpdated:<nil>} Alerts:{Alerts:[{Text:topology was created. Level:success}]}}
    topologies.go:48: Response: {Response:{Description:A topology with secondary parents Name:secondary-parents Nodes:[{Id:0 Cachegroup:parentCachegroup Parents:[] LastUpdated:<nil>} {Id:0 Cachegroup:cachegroup1 Parents:[0 2] LastUpdated:<nil>} {Id:0 Cachegroup:secondaryCachegroup Parents:[] LastUpdated:<nil>} {Id:0 Cachegroup:fallback1 Parents:[2 0] LastUpdated:<nil>} {Id:0 Cachegroup:fallback2 Parents:[2 0] LastUpdated:<nil>}] LastUpdated:<nil>} Alerts:{Alerts:[{Text:topology was created. Level:success}]}}
    topologies.go:48: Response: {Response:{Description:A 4-tier topology Name:4-tiers Nodes:[{Id:0 Cachegroup:parentCachegroup Parents:[] LastUpdated:<nil>} {Id:0 Cachegroup:parentCachegroup2 Parents:[0] LastUpdated:<nil>} {Id:0 Cachegroup:cachegroup1 Parents:[1] LastUpdated:<nil>} {Id:0 Cachegroup:secondaryCachegroup Parents:[1 0] LastUpdated:<nil>} {Id:0 Cachegroup:fallback1 Parents:[3] LastUpdated:<nil>}] LastUpdated:<nil>} Alerts:{Alerts:[{Text:topology was created. Level:success}]}}
    topologies.go:48: Response: {Response:{Description:This topology stems from 2 ancestors Name:forked-topology Nodes:[{Id:0 Cachegroup:topology-edge-cg-01 Parents:[2] LastUpdated:<nil>} {Id:0 Cachegroup:topology-edge-cg-02 Parents:[6] LastUpdated:<nil>} {Id:0 Cachegroup:topology-mid-cg-01 Parents:[3] LastUpdated:<nil>} {Id:0 Cachegroup:topology-mid-cg-02 Parents:[4] LastUpdated:<nil>} {Id:0 Cachegroup:topology-mid-cg-03 Parents:[5] LastUpdated:<nil>} {Id:0 Cachegroup:topology-mid-cg-04 Parents:[] LastUpdated:<nil>} {Id:0 Cachegroup:topology-mid-cg-05 Parents:[7] LastUpdated:<nil>} {Id:0 Cachegroup:topology-mid-cg-06 Parents:[8] LastUpdated:<nil>} {Id:0 Cachegroup:topology-mid-cg-07 Parents:[] LastUpdated:<nil>}] LastUpdated:<nil>} Alerts:{Alerts:[{Text:topology was created. Level:success}]}}
    topologies.go:48: Response: {Response:{Description:a topology Name:top-for-ds-req Nodes:[{Id:0 Cachegroup:dtrc1 Parents:[] LastUpdated:<nil>} {Id:0 Cachegroup:dtrc2 Parents:[0] LastUpdated:<nil>} {Id:0 Cachegroup:dtrc3 Parents:[0] LastUpdated:<nil>}] LastUpdated:<nil>} Alerts:{Alerts:[{Text:topology was created. Level:success}]}}
    topologies.go:48: Response: {Response:{Description:a topology Name:top-for-ds-req2 Nodes:[{Id:0 Cachegroup:dtrc1 Parents:[] LastUpdated:<nil>} {Id:0 Cachegroup:dtrc2 Parents:[0] LastUpdated:<nil>}] LastUpdated:<nil>} Alerts:{Alerts:[{Text:topology was created. Level:success}]}}
    topologies.go:48: Response: {Response:{Description:a topology Name:top-for-ds-cdn2 Nodes:[{Id:0 Cachegroup:cachegroup2 Parents:[] LastUpdated:<nil>}] LastUpdated:<nil>} Alerts:{Alerts:[{Text:topology was created. Level:success}]}}
    --- FAIL: TestT3cReload/reload_anything_in_trafficserver_dir (1.68s)
        t3c-reload_test.go:96: expected t3c to do a reload after adding a etc/trafficserver file, actual: INFO: config.go:536: 2021-11-23T16:28:54.104337569Z: set TSHome from the RPM config file  list '/opt/trafficserver'
            INFO: config.go:536: 2021-11-23T16:28:54.104367398Z: TSHome: /opt/trafficserver, TSConfigDir: /opt/trafficserver/etc/trafficserver
            DEBUG: config.go:585: 2021-11-23T16:28:54.104381241Z: LogLocationDebug: stderr
            DEBUG: config.go:586: 2021-11-23T16:28:54.104384647Z: LogLocationErr: stderr
            DEBUG: config.go:587: 2021-11-23T16:28:54.104387196Z: LogLocationInfo: stderr
            DEBUG: config.go:588: 2021-11-23T16:28:54.104389393Z: LogLocationWarn: stderr
            DEBUG: config.go:589: 2021-11-23T16:28:54.104391748Z: CacheHostName: atlanta-edge-03
            DEBUG: config.go:590: 2021-11-23T16:28:54.104395598Z: SvcManagement: SystemD
            DEBUG: config.go:591: 2021-11-23T16:28:54.104402466Z: Retries: 3
            DEBUG: config.go:592: 2021-11-23T16:28:54.104405147Z: ReverseProxyDisable: false
            DEBUG: config.go:593: 2021-11-23T16:28:54.104413374Z: SkipOSCheck: false
            DEBUG: config.go:594: 2021-11-23T16:28:54.104415846Z: TOInsecure: true
            DEBUG: config.go:595: 2021-11-23T16:28:54.104418362Z: TOTimeoutMS: 3000000000
            DEBUG: config.go:596: 2021-11-23T16:28:54.104421598Z: TOUser: admin
            DEBUG: config.go:597: 2021-11-23T16:28:54.104424328Z: TOPass: Pass len: '6'
            DEBUG: config.go:598: 2021-11-23T16:28:54.104427067Z: TOURL: https://to_server:443
            DEBUG: config.go:599: 2021-11-23T16:28:54.104429901Z: TSHome: /opt/trafficserver
            DEBUG: config.go:600: 2021-11-23T16:28:54.104432564Z: WaitForParents: false
            DEBUG: config.go:601: 2021-11-23T16:28:54.104435414Z: YumOptions:
            DEBUG: config.go:602: 2021-11-23T16:28:54.104438393Z: MaxmindLocation:
            INFO: t3c-apply.go:77: 2021-11-23T16:28:54.10444159Z: Trying to acquire app lock
            INFO: t3c-apply.go:85: 2021-11-23T16:28:54.10445706Z: Acquired app lock
            INFO: t3c-apply.go:92: 2021-11-23T16:28:54.106373961Z: Successfully ensured ATS config directory '/opt/trafficserver/etc/trafficserver' is a git repo
            DEBUG: util.go:405: 2021-11-23T16:28:54.113766094Z: the directory '/tmp/trafficcontrol-cache-config' already exists
            INFO: t3c-apply.go:122: 2021-11-23T16:28:54.113814587Z: 2021-11-23T16:28:54Z
            INFO: util.go:330: 2021-11-23T16:28:54.113832962Z: user check: report-only=false service-action=reload install-packages=false files=all user='root'
            WARNING: cmd.go:539: 2021-11-23T16:28:54.177457634Z: t3c-request start
            INFO: config.go:198: 2021-11-23T16:28:54.117747391Z: loading old config took 297ns
            INFO: config.go:175: 2021-11-23T16:28:54.117767733Z: using old config for IMS requests
            INFO: t3c-request.go:38: 2021-11-23T16:28:54.117772819Z: configuration initialized
            INFO: client.go:68: 2021-11-23T16:28:54.1177774Z: URL: 'https://to_server:443' User: 'admin' Pass len: '6'
            INFO: client.go:71: 2021-11-23T16:28:54.11778408Z: TO URL string: 'https://to_server:443'
            INFO: client.go:72: 2021-11-23T16:28:54.117786903Z: TO URL: 'https://to_server:443'
            INFO: client.go:85: 2021-11-23T16:28:54.17466019Z: toreqnew.New Logged into in to Traffic Ops '172.19.0.4:443'
            INFO: getdata.go:79: 2021-11-23T16:28:54.175030107Z: Getting data 'system-info'
            INFO: torequtil.go:60: 2021-11-23T16:28:54.176864567Z: GetRetry config_file_global_parameters retries 0 took 2ms
            WARNING: cmd.go:544: 2021-11-23T16:28:54.177521945Z: t3c-request end
            ERROR: torequest.go:668: 2021-11-23T16:28:54.177540203Z: Did not find tm.toolname!
            DEBUG: t3c-apply.go:129: 2021-11-23T16:28:54.17754537Z: toolname:
            DEBUG: torequest.go:728: 2021-11-23T16:28:54.177550262Z: Checking syncds state.
            WARNING: cmd.go:539: 2021-11-23T16:28:54.227885058Z: t3c-request start
            INFO: config.go:198: 2021-11-23T16:28:54.181542371Z: loading old config took 225ns
            INFO: config.go:175: 2021-11-23T16:28:54.181557072Z: using old config for IMS requests
            INFO: t3c-request.go:38: 2021-11-23T16:28:54.181561945Z: configuration initialized
            INFO: client.go:68: 2021-11-23T16:28:54.181565851Z: URL: 'https://to_server:443' User: 'admin' Pass len: '6'
            INFO: client.go:71: 2021-11-23T16:28:54.181571576Z: TO URL string: 'https://to_server:443'
            INFO: client.go:72: 2021-11-23T16:28:54.181574254Z: TO URL: 'https://to_server:443'
            INFO: client.go:85: 2021-11-23T16:28:54.222758428Z: toreqnew.New Logged into in to Traffic Ops '172.19.0.4:443'
            INFO: getdata.go:79: 2021-11-23T16:28:54.223149619Z: Getting data 'update-status'
            INFO: torequtil.go:60: 2021-11-23T16:28:54.227247601Z: GetRetry server_update_status_atlanta-edge-03 retries 0 took 4ms
            WARNING: cmd.go:544: 2021-11-23T16:28:54.227936745Z: t3c-request end
            ERROR: torequest.go:739: 2021-11-23T16:28:54.227985313Z: Traffic Ops is signaling that an update is waiting to be applied
            DEBUG: torequest.go:758: 2021-11-23T16:28:54.227995297Z: Processing with update: Traffic Ops server status &{HostName:atlanta-edge-03 UpdatePending:true RevalPending:false UseRevalPending:true HostId:1555 Status:REPORTED ParentPending:false ParentRevalPending:false} config wait-for-parents false
            DEBUG: torequest.go:271: 2021-11-23T16:28:54.228022239Z: Found REPORTED status from Traffic Ops.
            WARNING: cmd.go:539: 2021-11-23T16:28:54.280007488Z: t3c-request start
            INFO: config.go:198: 2021-11-23T16:28:54.231942317Z: loading old config took 261ns
            INFO: config.go:175: 2021-11-23T16:28:54.231961807Z: using old config for IMS requests
            INFO: t3c-request.go:38: 2021-11-23T16:28:54.231967949Z: configuration initialized
            INFO: client.go:68: 2021-11-23T16:28:54.231972988Z: URL: 'https://to_server:443' User: 'admin' Pass len: '6'
            INFO: client.go:71: 2021-11-23T16:28:54.231980873Z: TO URL string: 'https://to_server:443'
            INFO: client.go:72: 2021-11-23T16:28:54.231984367Z: TO URL: 'https://to_server:443'
            INFO: client.go:85: 2021-11-23T16:28:54.277539668Z: toreqnew.New Logged into in to Traffic Ops '172.19.0.4:443'
            INFO: getdata.go:79: 2021-11-23T16:28:54.277972952Z: Getting data 'statuses'
            INFO: torequtil.go:60: 2021-11-23T16:28:54.279401679Z: GetRetry statuses retries 0 took 1ms
            WARNING: cmd.go:544: 2021-11-23T16:28:54.280044857Z: t3c-request end
            DEBUG: util.go:405: 2021-11-23T16:28:54.280157621Z: the directory '/var/lib/trafficcontrol-cache-config/status' already exists
            INFO: t3c-apply.go:174: 2021-11-23T16:28:54.280168911Z: ======== Start processing packages  ========
            INFO: torequest.go:858: 2021-11-23T16:28:54.2801741Z: Calling ProcessPackages
            WARNING: cmd.go:539: 2021-11-23T16:28:54.334766901Z: t3c-request start
            INFO: config.go:198: 2021-11-23T16:28:54.284494594Z: loading old config took 208ns
            INFO: config.go:175: 2021-11-23T16:28:54.284509216Z: using old config for IMS requests
            INFO: t3c-request.go:38: 2021-11-23T16:28:54.28452516Z: configuration initialized
            INFO: client.go:68: 2021-11-23T16:28:54.284531579Z: URL: 'https://to_server:443' User: 'admin' Pass len: '6'
            INFO: client.go:71: 2021-11-23T16:28:54.284546046Z: TO URL string: 'https://to_server:443'
            INFO: client.go:72: 2021-11-23T16:28:54.284551255Z: TO URL: 'https://to_server:443'
            INFO: client.go:85: 2021-11-23T16:28:54.325932985Z: toreqnew.New Logged into in to Traffic Ops '172.19.0.4:443'
            INFO: getdata.go:79: 2021-11-23T16:28:54.326292071Z: Getting data 'packages'
            INFO: torequtil.go:60: 2021-11-23T16:28:54.33182228Z: GetRetry server-name-atlanta-edge-03 retries 0 took 6ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.334149664Z: GetRetry profile_ATS_EDGE_TIER_CACHE_parameters retries 0 took 2ms
            WARNING: cmd.go:544: 2021-11-23T16:28:54.334796811Z: t3c-request end
            INFO: torequest.go:864: 2021-11-23T16:28:54.334824122Z: ProcessPackages got [{Name:trafficserver Version:8.1.3-1.0d51dc3a7.el8.x86_64}]
            INFO: torequest.go:872: 2021-11-23T16:28:54.334834281Z: Processing package trafficserver-8.1.3-1.0d51dc3a7.el8.x86_64
            INFO: torequest.go:919: 2021-11-23T16:28:54.340157078Z:  Currently installed.
            DEBUG: torequest.go:932: 2021-11-23T16:28:54.340175375Z: number of packages requiring installation: 0
            DEBUG: torequest.go:936: 2021-11-23T16:28:54.340180789Z: number of packages requiring removal: 0
            DEBUG: t3c-apply.go:189: 2021-11-23T16:28:54.340184406Z: Preparing to fetch the config files for atlanta-edge-03, files: all, syncdsUpdate: UpdateTropsNeeded
            INFO: cmd.go:468: 2021-11-23T16:28:54.340296811Z: config cache bytes: 141361
            WARNING: cmd.go:539: 2021-11-23T16:28:54.462029957Z: t3c-request start
            INFO: config.go:198: 2021-11-23T16:28:54.348295068Z: loading old config took 3.015839ms
            INFO: config.go:175: 2021-11-23T16:28:54.348318377Z: using old config for IMS requests
            INFO: t3c-request.go:38: 2021-11-23T16:28:54.348324406Z: configuration initialized
            INFO: client.go:68: 2021-11-23T16:28:54.348328786Z: URL: 'https://to_server:443' User: 'admin' Pass len: '6'
            INFO: client.go:71: 2021-11-23T16:28:54.348336055Z: TO URL string: 'https://to_server:443'
            INFO: client.go:72: 2021-11-23T16:28:54.348339096Z: TO URL: 'https://to_server:443'
            INFO: client.go:85: 2021-11-23T16:28:54.40512048Z: toreqnew.New Logged into in to Traffic Ops '172.19.0.4:443'
            INFO: getdata.go:79: 2021-11-23T16:28:54.405692681Z: Getting data 'config'
            INFO: torequtil.go:60: 2021-11-23T16:28:54.408061001Z: GetRetry profile_global_parameters retries 0 took 2ms
            INFO: getdatacfg.go:195: 2021-11-23T16:28:54.408089961Z: Getting config: Global Params is modified, using new response
            INFO: getdatacfg.go:219: 2021-11-23T16:28:54.408098124Z: Traffic Ops proxy enabled, but GLOBAL Parameter 'tm.rev_proxy.url' missing or empty, not using proxy
            INFO: getdatacfg.go:591: 2021-11-23T16:28:54.408288335Z: Getting config: ServerCapabilities reqHdr map[If-Modified-Since:[Mon, 01 Jan 0001 00:00:01 UTC]]
            INFO: torequtil.go:60: 2021-11-23T16:28:54.414165299Z: GetRetry ds_regexes retries 0 took 6ms
            INFO: getdatacfg.go:650: 2021-11-23T16:28:54.414193682Z: Getting config: DeliveryServiceRegexes is modified, using new response
            INFO: getdatacfg.go:636: 2021-11-23T16:28:54.414206951Z: dsrF took 5.957989ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.416936629Z: GetRetry ds_capabilities_d_ retries 0 took 9ms
            INFO: getdatacfg.go:626: 2021-11-23T16:28:54.417014067Z: Getting config: DSRequiredCapabilities is modified, using new response
            INFO: getdatacfg.go:611: 2021-11-23T16:28:54.417056054Z: dscapsF took 8.715619ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.417172683Z: GetRetry topologies retries 0 took 9ms
            INFO: getdatacfg.go:743: 2021-11-23T16:28:54.417223467Z: Getting config: Topologies not modified, using old config
            INFO: getdatacfg.go:732: 2021-11-23T16:28:54.417262544Z: topologiesF took 8.943712ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.417397348Z: GetRetry config_file_parent.config_parameters retries 0 took 9ms
            INFO: getdatacfg.go:719: 2021-11-23T16:28:54.417417143Z: Getting config: ParentConfigParams not modified, using old config
            INFO: getdatacfg.go:708: 2021-11-23T16:28:54.417430285Z: parentConfigParamsF took 9.016225ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.417757085Z: GetRetry server_capabilities_s_ retries 0 took 9ms
            INFO: getdatacfg.go:601: 2021-11-23T16:28:54.417774425Z: Getting config: ServerCapabilities is modified, using new response
            INFO: getdatacfg.go:585: 2021-11-23T16:28:54.417790588Z: capsF took 9.530263ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.418289792Z: GetRetry cachegroups retries 0 took 10ms
            INFO: getdatacfg.go:573: 2021-11-23T16:28:54.418308067Z: Getting config: CacheGroups not modified, using old config
            INFO: getdatacfg.go:562: 2021-11-23T16:28:54.418318822Z: cfF took 10.051949ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.418456241Z: GetRetry config_file_remap.config_parameters retries 0 took 10ms
            INFO: getdatacfg.go:695: 2021-11-23T16:28:54.418471424Z: Getting config: RemapConfigParams not modified, using old config
            INFO: getdatacfg.go:684: 2021-11-23T16:28:54.418482852Z: remapConfigParamsF took 10.17085ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.419008896Z: GetRetry config_file_cachekey.config_parameters retries 0 took 11ms
            INFO: getdatacfg.go:674: 2021-11-23T16:28:54.419028302Z: Getting config: CacheKeyParams is modified, using new response
            INFO: getdatacfg.go:660: 2021-11-23T16:28:54.419040587Z: cacheKeyConfigParamsF took 10.675861ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.424865213Z: GetRetry servers retries 0 took 16ms
            INFO: getdatacfg.go:249: 2021-11-23T16:28:54.424877845Z: Getting config: Servers not modified, using old config
            INFO: torequtil.go:60: 2021-11-23T16:28:54.429589175Z: GetRetry profile_ATS_EDGE_TIER_CACHE retries 0 took 5ms
            INFO: getdatacfg.go:520: 2021-11-23T16:28:54.429611601Z: Getting config: Profile not modified, using old config
            INFO: getdatacfg.go:509: 2021-11-23T16:28:54.4296369Z: profileF took 4.720397ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.43064878Z: GetRetry profile_ATS_EDGE_TIER_CACHE_parameters retries 0 took 6ms
            INFO: getdatacfg.go:477: 2021-11-23T16:28:54.430673857Z: Getting config: ServerParams is modified, using new response
            INFO: getdatacfg.go:460: 2021-11-23T16:28:54.430687798Z: serverParamsF took 5.78873ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.431977538Z: GetRetry cdn_cdn1 retries 0 took 7ms
            INFO: getdatacfg.go:497: 2021-11-23T16:28:54.431998641Z: Getting config: CDN not modified, using old config
            INFO: getdatacfg.go:486: 2021-11-23T16:28:54.43201118Z: cdnF took 7.105928ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.432219435Z: GetRetry jobs_cdn_cdn1 retries 0 took 7ms
            INFO: getdatacfg.go:546: 2021-11-23T16:28:54.432233667Z: Getting config: Jobs is modified, using new response
            INFO: getdatacfg.go:532: 2021-11-23T16:28:54.432244392Z: jobsF took 7.348969ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.433758003Z: GetRetry cdn_156_deliveryservices retries 0 took 9ms
            INFO: getdatacfg.go:318: 2021-11-23T16:28:54.433773738Z: Getting config: DeliveryServices not modified, using old config
            INFO: getdatacfg.go:367: 2021-11-23T16:28:54.433815423Z: uriF took 2.247µs
            INFO: torequtil.go:60: 2021-11-23T16:28:54.436674087Z: GetRetry deliveryservice_servers_s_d__cdn_cdn1 retries 0 took 3ms
            INFO: getdatacfg.go:357: 2021-11-23T16:28:54.436701827Z: Getting config: DeliveryServiceServers is modified, using new response
            INFO: getdatacfg.go:341: 2021-11-23T16:28:54.436726263Z: dssF took 2.916962ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.438045807Z: GetRetry urlsigkeys_anymap-ds retries 0 took 4ms
            INFO: getdatacfg.go:438: 2021-11-23T16:28:54.438063334Z: Getting config: URLSigKeys[anymap-ds] is modified, using new response
            INFO: torequtil.go:60: 2021-11-23T16:28:54.441687948Z: GetRetry urlsigkeys_ds1 retries 0 took 4ms
            INFO: getdatacfg.go:438: 2021-11-23T16:28:54.441706841Z: Getting config: URLSigKeys[ds1] is modified, using new response
            INFO: torequtil.go:60: 2021-11-23T16:28:54.44527539Z: GetRetry urlsigkeys_ds1nat retries 0 took 4ms
            INFO: getdatacfg.go:438: 2021-11-23T16:28:54.445296355Z: Getting config: URLSigKeys[ds1nat] is modified, using new response
            INFO: torequtil.go:60: 2021-11-23T16:28:54.449124156Z: GetRetry urlsigkeys_ds2 retries 0 took 4ms
            INFO: getdatacfg.go:438: 2021-11-23T16:28:54.44914583Z: Getting config: URLSigKeys[ds2] is modified, using new response
            INFO: torequtil.go:60: 2021-11-23T16:28:54.449934946Z: GetRetry cdn_sslkeys_cdn1 retries 0 took 25ms
            INFO: getdatacfg.go:296: 2021-11-23T16:28:54.449955868Z: Getting config: SSLKeys is modified, using new response
            INFO: getdatacfg.go:279: 2021-11-23T16:28:54.449966584Z: sslF took 25.039897ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.452625057Z: GetRetry urlsigkeys_ds3 retries 0 took 3ms
            INFO: getdatacfg.go:438: 2021-11-23T16:28:54.452644415Z: Getting config: URLSigKeys[ds3] is modified, using new response
            INFO: torequtil.go:60: 2021-11-23T16:28:54.456295905Z: GetRetry urlsigkeys_ds-test-minor-versions retries 0 took 4ms
            INFO: getdatacfg.go:438: 2021-11-23T16:28:54.456315323Z: Getting config: URLSigKeys[ds-test-minor-versions] is modified, using new response
            INFO: torequtil.go:60: 2021-11-23T16:28:54.459750328Z: GetRetry urlsigkeys_msods1 retries 0 took 3ms
            INFO: getdatacfg.go:438: 2021-11-23T16:28:54.459767781Z: Getting config: URLSigKeys[msods1] is modified, using new response
            INFO: getdatacfg.go:408: 2021-11-23T16:28:54.459778278Z: urlF took 25.966308ms
            INFO: getdatacfg.go:305: 2021-11-23T16:28:54.459789716Z: dsF took 34.826011ms
            INFO: getdatacfg.go:236: 2021-11-23T16:28:54.459801484Z: serversF took 51.439262ms
            INFO: getdatacfg.go:176: 2021-11-23T16:28:54.459821676Z: GetTOData took 54.115694ms
            WARNING: cmd.go:544: 2021-11-23T16:28:54.462154471Z: t3c-request end
            WARNING: cmd.go:539: 2021-11-23T16:28:54.473053494Z: t3c-generate start
            ERROR: t3c-generate.go:53: 2021-11-23T16:28:54.467605766Z: Starting t3c-generate
            INFO: t3c-generate.go:64: 2021-11-23T16:28:54.46762339Z: reading Traffic Ops data from stdin
            WARNING: all.go:112: 2021-11-23T16:28:54.471646438Z: generating config files list: global tm.url parameter missing or empty! Setting empty in meta config!
            INFO: routing.go:40: 2021-11-23T16:28:54.471669476Z: GetConfigFile 'set_dscp_20.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.471682182Z: GetConfigFile set_dscp_20.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.47169271Z: GetConfigFile 'hdr_rw_first_ds-top.config'
            WARNING: all.go:112: 2021-11-23T16:28:54.471747752Z: getting config file 'hdr_rw_first_ds-top.config':
            INFO: routing.go:38: 2021-11-23T16:28:54.471754251Z: GetConfigFile hdr_rw_first_ds-top.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471759209Z: GetConfigFile 'remap.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.471816867Z: GetConfigFile remap.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471824095Z: GetConfigFile 'set_dscp_10.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.471828364Z: GetConfigFile set_dscp_10.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471831716Z: GetConfigFile 'non-empty-file.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.471841315Z: GetConfigFile non-empty-file.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471845386Z: GetConfigFile 'set_dscp_37.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.47184878Z: GetConfigFile set_dscp_37.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471852016Z: GetConfigFile 'set_dscp_24.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.471855167Z: GetConfigFile set_dscp_24.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471858617Z: GetConfigFile 'ssl_server_name.yaml'
            INFO: routing.go:38: 2021-11-23T16:28:54.471890832Z: GetConfigFile ssl_server_name.yaml took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471897038Z: GetConfigFile 'astats.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.47192277Z: GetConfigFile astats.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471927303Z: GetConfigFile 'set_dscp_22.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.471930736Z: GetConfigFile set_dscp_22.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471934383Z: GetConfigFile 'set_dscp_14.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.471937562Z: GetConfigFile set_dscp_14.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471940932Z: GetConfigFile 'set_dscp_12.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.471944325Z: GetConfigFile set_dscp_12.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471947627Z: GetConfigFile 'set_dscp_34.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.47195108Z: GetConfigFile set_dscp_34.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471955296Z: GetConfigFile 'plugin.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.471960126Z: GetConfigFile plugin.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471967844Z: GetConfigFile 'set_dscp_56.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.471971492Z: GetConfigFile set_dscp_56.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471975056Z: GetConfigFile 'empty-file.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.471980839Z: GetConfigFile empty-file.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471984166Z: GetConfigFile 'set_dscp_28.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.471987534Z: GetConfigFile set_dscp_28.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.471993575Z: GetConfigFile 'set_dscp_38.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472000573Z: GetConfigFile set_dscp_38.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472007291Z: GetConfigFile 'ip_allow.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472019711Z: GetConfigFile ip_allow.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472025151Z: GetConfigFile 'set_dscp_48.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472030794Z: GetConfigFile set_dscp_48.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472035662Z: GetConfigFile 'set_dscp_40.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472039328Z: GetConfigFile set_dscp_40.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472042693Z: GetConfigFile 'storage.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472049508Z: GetConfigFile storage.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472054284Z: GetConfigFile 'regex_revalidate.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472060736Z: GetConfigFile regex_revalidate.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472065566Z: GetConfigFile 'set_dscp_18.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472069098Z: GetConfigFile set_dscp_18.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472075721Z: GetConfigFile 'cache.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472083511Z: GetConfigFile cache.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472088595Z: GetConfigFile 'hosting.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472133438Z: GetConfigFile hosting.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472139431Z: GetConfigFile 'set_dscp_16.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472143334Z: GetConfigFile set_dscp_16.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472147509Z: GetConfigFile 'set_dscp_30.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472150905Z: GetConfigFile set_dscp_30.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472154709Z: GetConfigFile 'records.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472192371Z: GetConfigFile records.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.47219821Z: GetConfigFile 'set_dscp_36.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472202125Z: GetConfigFile set_dscp_36.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472205764Z: GetConfigFile 'volume.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472212825Z: GetConfigFile volume.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472217889Z: GetConfigFile 'set_dscp_0.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472221553Z: GetConfigFile set_dscp_0.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.47222535Z: GetConfigFile 'set_dscp_32.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472230922Z: GetConfigFile set_dscp_32.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472235704Z: GetConfigFile 'set_dscp_8.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472241689Z: GetConfigFile set_dscp_8.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472245266Z: GetConfigFile 'parent.config'
            WARNING: all.go:112: 2021-11-23T16:28:54.472381647Z: getting config file 'parent.config': duplicate origin! DS 'ds-top-req-cap2' and 'ds-top-req-cap' share origin 'http://example.org': skipping 'ds-top-req-cap2'!
            INFO: routing.go:38: 2021-11-23T16:28:54.472389809Z: GetConfigFile parent.config took 0s
            INFO: routing.go:40: 2021-11-23T16:28:54.472396688Z: GetConfigFile 'set_dscp_26.config'
            INFO: routing.go:38: 2021-11-23T16:28:54.472402044Z: GetConfigFile set_dscp_26.config took 0s
            INFO: plugin.go:129: 2021-11-23T16:28:54.472405545Z: plugins.ModifyFiles calling 0 plugins
            WARNING: cmd.go:544: 2021-11-23T16:28:54.473172762Z: t3c-generate end
            INFO: torequest.go:781: 2021-11-23T16:28:54.484370224Z:  ======== Start processing config files ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.484405166Z: About to process config file: /opt/trafficserver/etc/trafficserver/cache.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.484426163Z: the directory '/opt/trafficserver/etc/trafficserver/' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.48443557Z: ======== Start processing config file: cache.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.490616174Z: ======== End processing config file: cache.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.490631647Z: About to process config file: /opt/trafficserver/etc/trafficserver/records.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.490645802Z: the directory '/opt/trafficserver/etc/trafficserver/' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.490653302Z: ======== Start processing config file: records.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.49638438Z: ======== End processing config file: records.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.496400369Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_36.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.496414737Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.49642169Z: ======== Start processing config file: set_dscp_36.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.501912931Z: ======== End processing config file: set_dscp_36.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.501926228Z: About to process config file: /opt/trafficserver/etc/trafficserver/storage.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.501939534Z: the directory '/opt/trafficserver/etc/trafficserver/' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.501944933Z: ======== Start processing config file: storage.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.506894068Z: ======== End processing config file: storage.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.506905488Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_14.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.506917475Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.506922076Z: ======== Start processing config file: set_dscp_14.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.511124576Z: ======== End processing config file: set_dscp_14.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.511135396Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_22.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.511150136Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.511166498Z: ======== Start processing config file: set_dscp_22.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.515532899Z: ======== End processing config file: set_dscp_22.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.515543883Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_26.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.515560551Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.515570354Z: ======== Start processing config file: set_dscp_26.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.519675408Z: ======== End processing config file: set_dscp_26.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.519686682Z: About to process config file: /opt/trafficserver/etc/trafficserver/ip_allow.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.519697248Z: the directory '/opt/trafficserver/etc/trafficserver' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.519701069Z: ======== Start processing config file: ip_allow.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.524122089Z: ======== End processing config file: ip_allow.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.524135328Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_20.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.524147527Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.524153296Z: ======== Start processing config file: set_dscp_20.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.528746192Z: ======== End processing config file: set_dscp_20.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.528759808Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_40.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.528771837Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.528778289Z: ======== Start processing config file: set_dscp_40.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.532898975Z: ======== End processing config file: set_dscp_40.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.532913203Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_8.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.532927256Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.532941733Z: ======== Start processing config file: set_dscp_8.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.537240762Z: ======== End processing config file: set_dscp_8.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.537250848Z: About to process config file: /opt/trafficserver/etc/trafficserver/regex_revalidate.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.537259954Z: the directory '/opt/trafficserver/etc/trafficserver' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.537264283Z: ======== Start processing config file: regex_revalidate.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.541748947Z: ======== End processing config file: regex_revalidate.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.541760704Z: About to process config file: /opt/trafficserver/etc/trafficserver/remap.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.541771141Z: the directory '/opt/trafficserver/etc/trafficserver' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.541776119Z: ======== Start processing config file: remap.config ========
            WARNING: cmd.go:539: 2021-11-23T16:28:54.545335243Z: t3c-check-refs stderr start
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544598976Z: loaded plugin access_control.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544610308Z: loaded plugin acme.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544614087Z: loaded plugin astats_over_http.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544616958Z: loaded plugin authproxy.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544620976Z: loaded plugin background_fetch.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544623451Z: loaded plugin balancer.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544625783Z: loaded plugin buffer_upload.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544628351Z: loaded plugin cache_fill.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544630697Z: loaded plugin cache_promote.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544633844Z: loaded plugin cache_range_requests.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544639285Z: loaded plugin cachekey.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544641999Z: loaded plugin certifier.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544644522Z: loaded plugin collapsed_forwarding.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.54464696Z: loaded plugin combo_handler.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544650218Z: loaded plugin compress.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544653376Z: loaded plugin conf_remap.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544655914Z: loaded plugin custom_redirect.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544658456Z: loaded plugin escalate.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544660904Z: loaded plugin esi.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544663346Z: loaded plugin fq_pacing.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544665801Z: loaded plugin generator.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544668243Z: loaded plugin geoip_acl.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.54467068Z: loaded plugin header_freq.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544675367Z: loaded plugin header_normalize.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544678046Z: loaded plugin header_rewrite.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.54468055Z: loaded plugin healthchecks.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.54468302Z: loaded plugin hipes.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544689041Z: loaded plugin hook-trace.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544693318Z: loaded plugin inliner.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544695821Z: loaded plugin libloader.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544698256Z: loaded plugin metalink.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544700772Z: loaded plugin money_trace.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544703277Z: loaded plugin mp4.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544705853Z: loaded plugin multiplexer.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544708322Z: loaded plugin prefetch.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544710727Z: loaded plugin regex_remap.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544713078Z: loaded plugin regex_revalidate.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544715445Z: loaded plugin remap_purge.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544717774Z: loaded plugin remap_stats.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544720179Z: loaded plugin s3_auth.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544722625Z: loaded plugin server_push_preload.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544725119Z: loaded plugin slice.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544727609Z: loaded plugin ssl_cert_loader.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544733903Z: loaded plugin sslheaders.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544736773Z: loaded plugin stale_while_revalidate.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544739297Z: loaded plugin stats_over_http.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.54474182Z: loaded plugin stream_editor.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.54474441Z: loaded plugin system_stats.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544746854Z: loaded plugin tcpinfo.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.54474931Z: loaded plugin test_cppapi.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544751888Z: loaded plugin tls_bridge.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544754347Z: loaded plugin traffic_dump.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544756839Z: loaded plugin tslua.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544763005Z: loaded plugin tsmemcache.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544766262Z: loaded plugin uri_signing.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544769407Z: loaded plugin url_sig.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.544773867Z: loaded plugin xdebug.so
            DEBUG: t3c-check-refs.go:282: 2021-11-23T16:28:54.544781215Z: parsing: # DO NOT EDIT - Generated for atlanta-edge-03 by t3c-generate/0.3 from https://to_server:443 ips (172.19.0.4:443) on 2021-11-23T16:28:54.471666709Z
            DEBUG: t3c-check-refs.go:282: 2021-11-23T16:28:54.544787674Z: parsing: map	http://atlanta-edge-03.ds-top.test.cdn1.net/     http://origin.topology.example.net/ @plugin=header_rewrite.so @pparam=dscp/set_dscp_40.config @plugin=header_rewrite.so @pparam=hdr_rw_first_ds-top.config  @plugin=cachekey.so @pparam=--separator= @pparam=--remove-all-params=true @pparam=--remove-path=true @pparam=--capture-prefix-uri=/^([^?]*)/$1/ # topology 'mso-topology'
            DEBUG: t3c-check-refs.go:64: 2021-11-23T16:28:54.544796712Z: line: map	http://atlanta-edge-03.ds-top.test.cdn1.net/     http://origin.topology.example.net/ @plugin=header_rewrite.so @pparam=dscp/set_dscp_40.config @plugin=header_rewrite.so @pparam=hdr_rw_first_ds-top.config  @plugin=cachekey.so @pparam=--separator= @pparam=--remove-all-params=true @pparam=--remove-path=true @pparam=--capture-prefix-uri=/^([^?]*)/$1/ # topology 'mso-topology'
            DEBUG: t3c-check-refs.go:71: 2021-11-23T16:28:54.544840061Z: length: 15, fields: [map http://atlanta-edge-03.ds-top.test.cdn1.net/ http://origin.topology.example.net/ @plugin=header_rewrite.so @pparam=dscp/set_dscp_40.config @plugin=header_rewrite.so @pparam=hdr_rw_first_ds-top.config @plugin=cachekey.so @pparam=--separator= @pparam=--remove-all-params=true @pparam=--remove-path=true @pparam=--capture-prefix-uri=/^([^?]*)/$1/ # topology 'mso-topology']
            DEBUG: t3c-check-refs.go:89: 2021-11-23T16:28:54.544852071Z: Verified plugin 'header_rewrite.so', exists: false
            DEBUG: t3c-check-refs.go:180: 2021-11-23T16:28:54.544860139Z: verifying plugin file at /opt/trafficserver/libexec/trafficserver/header_rewrite.so
            INFO: t3c-check-refs.go:99: 2021-11-23T16:28:54.544868649Z: then plugin DSO 'header_rewrite.so' in remap.config on line '1' has been verified
            INFO: t3c-check-refs.go:125: 2021-11-23T16:28:54.544910605Z: the plugin config file 'dscp/set_dscp_40.config' on line '1' of remap.config has been verified
            DEBUG: t3c-check-refs.go:89: 2021-11-23T16:28:54.544918464Z: Verified plugin 'header_rewrite.so', exists: true
            INFO: t3c-check-refs.go:99: 2021-11-23T16:28:54.544922909Z: then plugin DSO 'header_rewrite.so' in remap.config on line '1' has been verified
            INFO: t3c-check-refs.go:125: 2021-11-23T16:28:54.544939819Z: the plugin config file 'hdr_rw_first_ds-top.config' on line '1' of remap.config has been verified
            DEBUG: t3c-check-refs.go:89: 2021-11-23T16:28:54.544945228Z: Verified plugin 'cachekey.so', exists: false
            DEBUG: t3c-check-refs.go:180: 2021-11-23T16:28:54.54494991Z: verifying plugin file at /opt/trafficserver/libexec/trafficserver/cachekey.so
            INFO: t3c-check-refs.go:99: 2021-11-23T16:28:54.544955659Z: then plugin DSO 'cachekey.so' in remap.config on line '1' has been verified
            DEBUG: t3c-check-refs.go:282: 2021-11-23T16:28:54.54499602Z: parsing: map	https://atlanta-edge-03.ds-top.test.cdn1.net/     http://origin.topology.example.net/ @plugin=header_rewrite.so @pparam=dscp/set_dscp_40.config @plugin=header_rewrite.so @pparam=hdr_rw_first_ds-top.config  @plugin=cachekey.so @pparam=--separator= @pparam=--remove-all-params=true @pparam=--remove-path=true @pparam=--capture-prefix-uri=/^([^?]*)/$1/ # topology 'mso-topology'
            DEBUG: t3c-check-refs.go:64: 2021-11-23T16:28:54.545001288Z: line: map	https://atlanta-edge-03.ds-top.test.cdn1.net/     http://origin.topology.example.net/ @plugin=header_rewrite.so @pparam=dscp/set_dscp_40.config @plugin=header_rewrite.so @pparam=hdr_rw_first_ds-top.config  @plugin=cachekey.so @pparam=--separator= @pparam=--remove-all-params=true @pparam=--remove-path=true @pparam=--capture-prefix-uri=/^([^?]*)/$1/ # topology 'mso-topology'
            DEBUG: t3c-check-refs.go:71: 2021-11-23T16:28:54.545016689Z: length: 15, fields: [map https://atlanta-edge-03.ds-top.test.cdn1.net/ http://origin.topology.example.net/ @plugin=header_rewrite.so @pparam=dscp/set_dscp_40.config @plugin=header_rewrite.so @pparam=hdr_rw_first_ds-top.config @plugin=cachekey.so @pparam=--separator= @pparam=--remove-all-params=true @pparam=--remove-path=true @pparam=--capture-prefix-uri=/^([^?]*)/$1/ # topology 'mso-topology']
            DEBUG: t3c-check-refs.go:89: 2021-11-23T16:28:54.545022741Z: Verified plugin 'header_rewrite.so', exists: true
            INFO: t3c-check-refs.go:99: 2021-11-23T16:28:54.54502697Z: then plugin DSO 'header_rewrite.so' in remap.config on line '2' has been verified
            INFO: t3c-check-refs.go:125: 2021-11-23T16:28:54.545047852Z: the plugin config file 'dscp/set_dscp_40.config' on line '2' of remap.config has been verified
            DEBUG: t3c-check-refs.go:89: 2021-11-23T16:28:54.545053508Z: Verified plugin 'header_rewrite.so', exists: true
            INFO: t3c-check-refs.go:99: 2021-11-23T16:28:54.545057629Z: then plugin DSO 'header_rewrite.so' in remap.config on line '2' has been verified
            INFO: t3c-check-refs.go:125: 2021-11-23T16:28:54.545072333Z: the plugin config file 'hdr_rw_first_ds-top.config' on line '2' of remap.config has been verified
            DEBUG: t3c-check-refs.go:89: 2021-11-23T16:28:54.545078888Z: Verified plugin 'cachekey.so', exists: true
            INFO: t3c-check-refs.go:99: 2021-11-23T16:28:54.545083061Z: then plugin DSO 'cachekey.so' in remap.config on line '2' has been verified
            INFO: t3c-check-refs.go:309: 2021-11-23T16:28:54.545122306Z: All configured plugins have successfully been verified
            WARNING: cmd.go:544: 2021-11-23T16:28:54.545472266Z: t3c-check-refs stderr end
            INFO: torequest.go:217: 2021-11-23T16:28:54.545477341Z: Successfully verified plugins used by 'remap.config'
            INFO: torequest.go:235: 2021-11-23T16:28:54.550190412Z: ======== End processing config file: remap.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.550204217Z: About to process config file: /opt/trafficserver/etc/trafficserver/ssl_server_name.yaml, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.550214658Z: the directory '/opt/trafficserver/etc/trafficserver' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.550226754Z: ======== Start processing config file: ssl_server_name.yaml ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.554603151Z: ======== End processing config file: ssl_server_name.yaml for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.554613987Z: About to process config file: /opt/trafficserver/etc/trafficserver/parent.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.554624142Z: the directory '/opt/trafficserver/etc/trafficserver/' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.554629244Z: ======== Start processing config file: parent.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.558875878Z: ======== End processing config file: parent.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.558888415Z: About to process config file: /opt/trafficserver/etc/trafficserver/plugin.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.558900495Z: the directory '/opt/trafficserver/etc/trafficserver/' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.558907661Z: ======== Start processing config file: plugin.config ========
            WARNING: cmd.go:539: 2021-11-23T16:28:54.562323503Z: t3c-check-refs stderr start
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561852062Z: loaded plugin access_control.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561864081Z: loaded plugin acme.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561867945Z: loaded plugin astats_over_http.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561870786Z: loaded plugin authproxy.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561873278Z: loaded plugin background_fetch.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561875702Z: loaded plugin balancer.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561878138Z: loaded plugin buffer_upload.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561880487Z: loaded plugin cache_fill.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561884488Z: loaded plugin cache_promote.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561892411Z: loaded plugin cache_range_requests.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561898291Z: loaded plugin cachekey.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561901255Z: loaded plugin certifier.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561903965Z: loaded plugin collapsed_forwarding.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561906524Z: loaded plugin combo_handler.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561909859Z: loaded plugin compress.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.56191242Z: loaded plugin conf_remap.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561914928Z: loaded plugin custom_redirect.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561917401Z: loaded plugin escalate.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561919941Z: loaded plugin esi.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561922495Z: loaded plugin fq_pacing.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561925008Z: loaded plugin generator.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561927455Z: loaded plugin geoip_acl.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561930026Z: loaded plugin header_freq.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561933217Z: loaded plugin header_normalize.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561935724Z: loaded plugin header_rewrite.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561938212Z: loaded plugin healthchecks.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561942485Z: loaded plugin hipes.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561948273Z: loaded plugin hook-trace.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561951256Z: loaded plugin inliner.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561954105Z: loaded plugin libloader.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561956712Z: loaded plugin metalink.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561959158Z: loaded plugin money_trace.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561961634Z: loaded plugin mp4.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561964189Z: loaded plugin multiplexer.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561966636Z: loaded plugin prefetch.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561969012Z: loaded plugin regex_remap.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561971369Z: loaded plugin regex_revalidate.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561973738Z: loaded plugin remap_purge.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561976227Z: loaded plugin remap_stats.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561978679Z: loaded plugin s3_auth.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.56198114Z: loaded plugin server_push_preload.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561983671Z: loaded plugin slice.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561986154Z: loaded plugin ssl_cert_loader.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561988594Z: loaded plugin sslheaders.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561991058Z: loaded plugin stale_while_revalidate.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561993573Z: loaded plugin stats_over_http.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.561997498Z: loaded plugin stream_editor.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.562000031Z: loaded plugin system_stats.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.562002484Z: loaded plugin tcpinfo.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.562004951Z: loaded plugin test_cppapi.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.562007517Z: loaded plugin tls_bridge.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.562010123Z: loaded plugin traffic_dump.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.562012611Z: loaded plugin tslua.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.562018585Z: loaded plugin tsmemcache.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.562022227Z: loaded plugin uri_signing.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.56202541Z: loaded plugin url_sig.so
            DEBUG: t3c-check-refs.go:199: 2021-11-23T16:28:54.562030225Z: loaded plugin xdebug.so
            DEBUG: t3c-check-refs.go:282: 2021-11-23T16:28:54.562038121Z: parsing: # DO NOT EDIT - Generated for atlanta-edge-03 by t3c-generate/0.3 from https://to_server:443 ips (172.19.0.4:443) on 2021-11-23T16:28:54.471666709Z
            DEBUG: t3c-check-refs.go:282: 2021-11-23T16:28:54.562044462Z: parsing:
            DEBUG: t3c-check-refs.go:64: 2021-11-23T16:28:54.562047835Z: line:
            DEBUG: t3c-check-refs.go:71: 2021-11-23T16:28:54.562066024Z: length: 1, fields: []
            INFO: t3c-check-refs.go:309: 2021-11-23T16:28:54.562098488Z: All configured plugins have successfully been verified
            WARNING: cmd.go:544: 2021-11-23T16:28:54.562399074Z: t3c-check-refs stderr end
            INFO: torequest.go:217: 2021-11-23T16:28:54.56240384Z: Successfully verified plugins used by 'plugin.config'
            INFO: torequest.go:235: 2021-11-23T16:28:54.56685486Z: ======== End processing config file: plugin.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.566868751Z: About to process config file: /opt/trafficserver/etc/trafficserver/volume.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.566879979Z: the directory '/opt/trafficserver/etc/trafficserver/' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.56688471Z: ======== Start processing config file: volume.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.571236804Z: ======== End processing config file: volume.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.571249522Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_18.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.571267539Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.571274636Z: ======== Start processing config file: set_dscp_18.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.576191544Z: ======== End processing config file: set_dscp_18.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.576206684Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_34.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.576225789Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.576234033Z: ======== Start processing config file: set_dscp_34.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.580741248Z: ======== End processing config file: set_dscp_34.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.580755447Z: About to process config file: /opt/trafficserver/etc/trafficserver/hdr_rw_first_ds-top.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.580771174Z: the directory '/opt/trafficserver/etc/trafficserver' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.580780202Z: ======== Start processing config file: hdr_rw_first_ds-top.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.585222444Z: ======== End processing config file: hdr_rw_first_ds-top.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.585233433Z: About to process config file: /opt/trafficserver/etc/trafficserver/empty-file.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.58524901Z: the directory '/opt/trafficserver/etc/trafficserver/' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.585255686Z: ======== Start processing config file: empty-file.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.589675775Z: ======== End processing config file: empty-file.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.589689019Z: About to process config file: /opt/trafficserver/etc/trafficserver/hosting.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.589705084Z: the directory '/opt/trafficserver/etc/trafficserver/' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.589712643Z: ======== Start processing config file: hosting.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.593998527Z: ======== End processing config file: hosting.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.594015706Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_0.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.594030606Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.594039173Z: ======== Start processing config file: set_dscp_0.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.598540536Z: ======== End processing config file: set_dscp_0.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.598552377Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_30.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.598563058Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.598567013Z: ======== Start processing config file: set_dscp_30.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.60284386Z: ======== End processing config file: set_dscp_30.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.602856163Z: About to process config file: /opt/trafficserver/etc/trafficserver/astats.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.602869417Z: the directory '/opt/trafficserver/etc/trafficserver' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.60287922Z: ======== Start processing config file: astats.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.607031055Z: ======== End processing config file: astats.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.607046463Z: About to process config file: /opt/trafficserver/etc/trafficserver/non-empty-file.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.607064609Z: the directory '/opt/trafficserver/etc/trafficserver/' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.607072477Z: ======== Start processing config file: non-empty-file.config ========
            INFO: cmd.go:300: 2021-11-23T16:28:54.611089177Z: file '/opt/trafficserver/etc/trafficserver/non-empty-file.config' changes begin
            INFO: cmd.go:300: 2021-11-23T16:28:54.611098718Z: diff: -bar
            INFO: cmd.go:300: 2021-11-23T16:28:54.611102107Z: diff: +
            INFO: cmd.go:300: 2021-11-23T16:28:54.611104916Z: file '/opt/trafficserver/etc/trafficserver/non-empty-file.config' changes end
            INFO: torequest.go:235: 2021-11-23T16:28:54.611107908Z: ======== End processing config file: non-empty-file.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.611112386Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_10.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.611121652Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.611126263Z: ======== Start processing config file: set_dscp_10.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.615052263Z: ======== End processing config file: set_dscp_10.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.615063127Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_12.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.615074111Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.615079419Z: ======== Start processing config file: set_dscp_12.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.619528563Z: ======== End processing config file: set_dscp_12.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.619540888Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_16.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.619552357Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.619557596Z: ======== Start processing config file: set_dscp_16.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.62383347Z: ======== End processing config file: set_dscp_16.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.623845227Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_28.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.623856106Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.623861545Z: ======== Start processing config file: set_dscp_28.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.628302463Z: ======== End processing config file: set_dscp_28.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.628315136Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_37.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.628325907Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.628331356Z: ======== Start processing config file: set_dscp_37.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.632548983Z: ======== End processing config file: set_dscp_37.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.632560529Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_38.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.632580903Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.632586709Z: ======== Start processing config file: set_dscp_38.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.636910185Z: ======== End processing config file: set_dscp_38.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.636922412Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_24.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.63693276Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.636938069Z: ======== Start processing config file: set_dscp_24.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.641413018Z: ======== End processing config file: set_dscp_24.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.641427133Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_32.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.64144225Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.641449486Z: ======== Start processing config file: set_dscp_32.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.645827283Z: ======== End processing config file: set_dscp_32.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.645842261Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_48.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.645855313Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.645861347Z: ======== Start processing config file: set_dscp_48.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.649985834Z: ======== End processing config file: set_dscp_48.config for service: trafficserver ========
            DEBUG: torequest.go:805: 2021-11-23T16:28:54.649997857Z: About to process config file: /opt/trafficserver/etc/trafficserver/dscp/set_dscp_56.config, service: trafficserver
            DEBUG: util.go:405: 2021-11-23T16:28:54.650017318Z: the directory '/opt/trafficserver/etc/trafficserver/dscp' already exists
            DEBUG: torequest.go:203: 2021-11-23T16:28:54.650023592Z: ======== Start processing config file: set_dscp_56.config ========
            INFO: torequest.go:235: 2021-11-23T16:28:54.654347822Z: ======== End processing config file: set_dscp_56.config for service: trafficserver ========
            DEBUG: torequest.go:835: 2021-11-23T16:28:54.654363865Z: All Prereqs passed for replacing non-empty-file.config on disk with that in Traffic Ops.
            INFO: torequest.go:474: 2021-11-23T16:28:54.654369871Z: Writing temp file '/opt/trafficserver/etc/trafficserver/non-empty-file.config.tmp' with file mode: '0644'
            INFO: torequest.go:485: 2021-11-23T16:28:54.654438174Z: Copying temp file '/opt/trafficserver/etc/trafficserver/non-empty-file.config.tmp' to real '/opt/trafficserver/etc/trafficserver/non-empty-file.config'
            DEBUG: torequest.go:515: 2021-11-23T16:28:54.654467183Z: Reload state after non-empty-file.config: remap.config: false reload: false restart: false ntpd: false sysctl: false
            DEBUG: torequest.go:517: 2021-11-23T16:28:54.65447436Z: Setting change applied for 'non-empty-file.config'
            INFO: torequest.go:845: 2021-11-23T16:28:54.65447936Z: Final state: remap.config: false reload: false restart: false ntpd: false sysctl: false
            INFO: t3c-apply.go:282: 2021-11-23T16:28:54.654484897Z: maxmindlocation is empty, not checking for DB update
            INFO: cmd.go:338: 2021-11-23T16:28:54.654489298Z: t3c-check-reload calling with pluginPackagesInstalled '[]' changedConfigFiles '[/opt/trafficserver/etc/trafficserver/non-empty-file.config]'
            INFO: torequest.go:1046: 2021-11-23T16:28:54.658520316Z: t3c-check-reload returned 'reload'
            INFO: torequest.go:1095: 2021-11-23T16:28:54.687140576Z: ATS configuration has changed, Running 'traffic_ctl config reload' now.
            INFO: torequest.go:1105: 2021-11-23T16:28:54.691064397Z: ATS 'traffic_ctl config reload' was successful
            INFO: torequest.go:582: 2021-11-23T16:28:54.69108247Z: IsPackageInstalled 'teakd' not found in cache, querying rpm
            INFO: torequest.go:595: 2021-11-23T16:28:54.696358848Z: IsPackageInstalled 'teakd' not found in rpm, adding 'teakd'=false to cache
            WARNING: cmd.go:539: 2021-11-23T16:28:54.764330061Z: t3c-request start
            INFO: config.go:198: 2021-11-23T16:28:54.701050922Z: loading old config took 301ns
            INFO: config.go:175: 2021-11-23T16:28:54.701073051Z: using old config for IMS requests
            INFO: t3c-request.go:38: 2021-11-23T16:28:54.701078917Z: configuration initialized
            INFO: client.go:68: 2021-11-23T16:28:54.701083115Z: URL: 'https://to_server:443' User: 'admin' Pass len: '6'
            INFO: client.go:71: 2021-11-23T16:28:54.701093128Z: TO URL string: 'https://to_server:443'
            INFO: client.go:72: 2021-11-23T16:28:54.701096438Z: TO URL: 'https://to_server:443'
            INFO: client.go:85: 2021-11-23T16:28:54.752190241Z: toreqnew.New Logged into in to Traffic Ops '172.19.0.4:443'
            INFO: getdata.go:79: 2021-11-23T16:28:54.753184355Z: Getting data 'update-status'
            INFO: torequtil.go:60: 2021-11-23T16:28:54.763194892Z: GetRetry server_update_status_atlanta-edge-03 retries 0 took 10ms
            WARNING: cmd.go:544: 2021-11-23T16:28:54.764408357Z: t3c-request end
            ERROR: torequest.go:1144: 2021-11-23T16:28:54.764461611Z: Traffic Ops requires an update and it was applied successfully.  Clearing update state in Traffic Ops.
            WARNING: cmd.go:539: 2021-11-23T16:28:54.851654051Z: t3c-update start
            INFO: t3c-update.go:45: 2021-11-23T16:28:54.774461524Z: configuration initialized
            INFO: client.go:68: 2021-11-23T16:28:54.774503414Z: URL: 'https://to_server:443' User: 'admin' Pass len: '6'
            INFO: client.go:71: 2021-11-23T16:28:54.774520884Z: TO URL string: 'https://to_server:443'
            INFO: client.go:72: 2021-11-23T16:28:54.774528655Z: TO URL: 'https://to_server:443'
            INFO: client.go:85: 2021-11-23T16:28:54.843341666Z: toreqnew.New Logged into in to Traffic Ops '172.19.0.4:443'
            INFO: torequtil.go:60: 2021-11-23T16:28:54.846752492Z: GetRetry set_server_update_status_atlanta-edge-03 retries 0 took 3ms
            INFO: torequtil.go:60: 2021-11-23T16:28:54.850942772Z: GetRetry server_update_status_atlanta-edge-03 retries 0 took 4ms
            INFO: t3c-update.go:79: 2021-11-23T16:28:54.850954368Z: Update successfully completed
            WARNING: cmd.go:544: 2021-11-23T16:28:54.851708607Z: t3c-update end
            INFO: cmd.go:253: 2021-11-23T16:28:54.851714015Z: t3c-update succeeded
            INFO: torequest.go:1172: 2021-11-23T16:28:54.85171975Z: Traffic Ops has been updated.
    --- FAIL: TestT3cReload/reload_no_change (1.65s)
        t3c-reload_test.go:119: not a real error
    --- FAIL: TestT3cReload/reval_calls_reload (1.51s)
        t3c-reload_test.go:126: not a real error
    types.go:58: ---- DeleteTestTypes ----
User: traffic_ops, Password: len 6, Hostname: db, DB: traffic_ops
User: traffic_ops, Password: len 6, Hostname: db, DB: traffic_ops
User: traffic_ops, Password: len 6, Hostname: db, DB: traffic_ops
User: traffic_ops, Password: len 6, Hostname: db, DB: traffic_ops
User: traffic_ops, Password: len 6, Hostname: db, DB: traffic_ops
User: traffic_ops, Password: len 6, Hostname: db, DB: traffic_ops
User: traffic_ops, Password: len 6, Hostname: db, DB: traffic_ops
User: traffic_ops, Password: len 6, Hostname: db, DB: traffic_ops
FAIL
exit status 1
FAIL	github.com/apache/trafficcontrol/cache-config/testing/ort-tests	224.505s
```

</details>

(Output from unrelated tests stripped, some spammy lines for creating servers/statuses/profiles/parameters also removed)

Multiline output for a sub-test is indented on each subsequent line to make it visually associated with the related sub-test - I think this is much more visually apparent than triple quotes could hope to be. That being said, I don't feel strongly about it. I think it's unnecessary to add them, but I don't think it has much negative impact if done anyway, so if you still think it should have them I'll put them back in.

Reply:
Ok yeah, if it's indented, that seems good enough.